packages/headless/src/api/analytics/product-recommendations-analytics.ts

```diff
@@ -17,6 +18,10 @@ export class ProductRecommendationAnalyticsProvider
     private state: StateNeededByProductRecommendationsAnalyticsProvider
   ) {}
 
+  public getLanguage() {
```

Comment:
I see that this method and it's implementation are duplicated on `AnalyticsProvider` and `ProductRecommendationAnalyticsProvider`. Let's say tomorrow we add a third provider.

Is `getLanguage` expected to be available on this provider?
Should `getLanguage` be a part of the `SearchPageClientProvider` interface that all providers implement?

Reply:
> Should getLanguage be a part of the SearchPageClientProvider interface that all providers implement?

It is, yes.

That's why I needed to implement it for those two.